networkx/convert_matrix.py

```diff
@@ -384,7 +388,7 @@ def from_pandas_edgelist(
     AtlasView({'A': {'weight': 3, 'color': 'red'}, 'D': {'weight': 6, 'color': 'blue'}})
 
 
-    """
+    """  # noqa: E501
```

Comment:
Possibly. My local flake8 tells me the line is too long. No big deal, I'll just default to using what the NetworkX CI shows. That said, providing a locally-available config file, I think, will be very helpful for developers. Waiting for the CI to start up only to find out that I missed one or two code formatting errors because of a mismatched config can build up frustration in the long-run. (For example, I always black configured in VSCode to format to 79 char line lengths, as that's optimal for using VSCode on a 15" laptop or larger to see two source files side-by-side, but this conflicts with some projects.)

Reply:
NetworkX is using the default settings for black.  The benefit of using black is diminished if we are going each use different settings for black.  This PR changes line lengths for your setting of black.  If I edit this file later, black will reformat those lines to the default length.